uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/request-ctrl_test.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-env amd, jasmine */
```

Comment:
Despite of the fact that I actually agree with @reneolivo in general, in that particular case we should move to `leave-request-ctrl_test` because the parent controller is indeed never used directly. These files were built a long time ago and I personally don't like how they are structured, but we don't want to mess with them at least in this particular PR.